src/styles/index.css

```diff
@@ -15,7 +15,7 @@
 }
 
 h1 {
-  @apply text-lg;
+  @apply text-xl;
 }
```

Comment:
**thought:** I actually like tailwind's hard reset on everything and don't mind setting things like this on component level. I've noticed you're already modifying font size in prose config.

Reply:
Is there a way for us to bump up the typographic scale across the whole site?
The default prose one doesn't have enough size distinction between h1, h2, h3. Want the h1's across the whole site to be bigger than prose sets by default and seemed like the easiest way to do it?

Reply:
I'd prefer tailwind's reset applied to everything by default. utility-first approach. h1's usually aren't part of prose wrapper and can be styled individually as we see fit.

Reply:
Yup no worries that works, updated with 5bf101c and be8378a. Moved the size adjustments to the prose config so it'll only apply to articles and won't mess up font sizes across the site as a whole.